preprocessing/processing.py

```diff
@@ -0,0 +1,272 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
I don't know what the recommended format for the header is, with the licensing info here, but can we include a comment at/near the beginning of the file describing what you are doing in this file (e.g. are you converting all the datasets to a common format? What is that format?)

Reply:
I added a small description paragraph after the license information. Do you think that looks okay?

Reply:
Looks great, thanks!